src/app/user-profile-search/checkboxes-group/checkboxes-group.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FilterValue } from '../filter-value';
+
+@Component({
+  selector: 'app-checkboxes-group',
+  templateUrl: './checkboxes-group.component.html',
+  styleUrls: ['./checkboxes-group.component.scss']
+})
+export class CheckboxesGroupComponent implements OnInit {
```

Comment:
I'd recommend to use a consistent naming. This component should be called `FilterGroupComponent` because its values are also typed `FilterValue` and this really describes its purpose.

Reply:
Perfect name! Thank you 😊